contrib/azure/azure-coreos-cluster

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
+
+from azure import *
+from azure.servicemanagement import *
+import argparse
+import urllib2
+import time
+import base64
+import os
+import subprocess
+
+parser = argparse.ArgumentParser(description='Create a CoreOS cluster on Microsoft Azure.')
+parser.add_argument('--version', action='version', version='azure-coreos-cluster 0.1')
```

Comment:
I don't want to change things that would make it hard to pull in upstream changes from @chanezon: https://github.com/chanezon/azure-linux/blob/master/coreos/cluster/azure-coreos-cluster